__init__.py

```diff
@@ -1,8 +1,10 @@
+from __version__ import VERSION
+
 bl_info = {
     "name": "Frontier's Cobra Engine Formats",
     "author": "Harlequinz Ego, HENDRIX et al.",
     "blender": (4, 0, 0),
-    "version": (3, 0, 2),
+    "version": VERSION.split("."),
```

Comment:
this does not work, it has to be hardcoded for blender as this dict is parsed and not interpreted by python.

Reply:
Can I let `increment_version` hook also change this line?

Reply:
Clever thought, I had not thought of that. Should be possible, I think!

Reply:
2379a088833bef427195691e1652db904c8c15e7 now the package version defined in `pyproject.toml` is the single source of truth, `__version__.py` and version of blender plugin are auto-synced with it.

Reply:
oh we don't have any test added to confirm the version works right?
Now I'm curious to know when are we even going to bump the version since we'd have this feature at hand! 🥇

Reply:
It's a bit hard to write test for this hook but I did test it manually.

About the semantic version I have the same question. I assume it's only used by the blender plugin and nobody is really enforcing a versioning strategy. Because majority of the users download code zip to use this tool, so commit hash is the real version.

Reply:
Thanks @zachguo, looks good to me too!

Reply:
Congratulations on your first commit in this project!